modules/admin/src/test/kotlin/jp/co/soramitsu/iroha2/ClientTest.kt

```diff
@@ -29,4 +29,21 @@ class ClientTest {
         val status = client.status()
         println(status["blocks"] == 1)
     }
+
+    @Test
+    @WithIroha
+    fun metrics(): Unit = runBlocking {
+        val metrics = client.metrics()
+        assert(metrics.isNotEmpty())
+    }
+
+    @Test
+    @WithIroha
+    fun configure(): Unit = runBlocking {
+        val valueConfig = client.valueConfig()
```

Comment:
please check if argument was provided it also works `client.valueConfig(listOf(...smth...))`